_extras/design.md

```diff
@@ -116,7 +116,117 @@ How do I...
 
 ## Stage 2 - Learning Plan
 
-*To Do: point-form outline of the lesson episodes with time estimates for teaching and exercises.*
+### Introductions and Learning Objectives
+
+*   Teaching: 15 min
+*   Exercises: 0 min (accounted for in teaching time - no separate exercise)
+    *   Ensure setup issues resolved
+    *   Assess how much `dplyr` review will be necessary
+
+### Data Management in R
+
+*   Teaching: 10 min
+*   Exercises: 5 min
+    *   Import Gapminder data (`read.csv()`)
+    *   Gapminder data structure (`str()`, `head()`)
+
+### Data Structures
+
+*   Teaching: 5 min
+*   Exercises: 5 min
+    *   Variables and categories
+    *   Values and replicates
+    *   Absolute values vs. relative values
+
+### `dplyr` Recap
+
+*   Teaching: 10 min
+*   Exercises: 5 min
+    *   `select()`
+    *   `filter()`
+    *   `group_by()`
+    *   `summarize()`
+
+### Scientific Questions and Hypotheses
+
+*   Teaching: 10 min
+*   Exercises: 5 min
+    *   Define: distribution, relationship, comparison, composition
```

Comment:
Hard to give feedback on a short timescale in class to that - how about "match something from column A to something from column B" ?